src/odemis/driver/xt_client.py

```diff
@@ -2377,10 +2375,8 @@ def _moveTo(self, future: futures.Future, pos: Dict[str, float], rel: bool = Fal
                 # The stage is not moving anymore, however in some rare cases, the server still reports the old
                 # position for a short while (typically < 1s). This can cause all sorts of confusion in the calling
                 # code, as it would seem that the move didn't have any effect, and later on, after the position
-                # is eventually updated that the stage has moved unexpectedly. So, wait until the reported position is
-                # (1) not identical to the starting position and (2) not too far from the target position.
+                # is eventually updated that the stage has moved unexpectedly. So, wait until close to the target position.
 
-                # The stage is not moving anymore, but we still want to wait until the position has been updated
                 # TODO: base the timeout on stage movement time estimation, needs to be checked on hardware
                 wait_stage_move = 10  # s
```

Comment:
10 seconds still feels quite long for the default timeout

Reply:
A timeout should be something that feels really long. It should be there as better behaviour than "waiting forever". Here, in this function, normally the new stage position is immediately provided. In some cases, we need to wait a little bit that it's provided (due to a bug in the TFS software). If that also fails, then we give up after 10s. I think for very long moves, there could be merit to wait *longer*.

Anyway, let's not change this part now. I'd rather do this in a separate task/PR.